### PR TITLE
Rename DpslTypes to TelemetryUnion

### PR DESCRIPTION
**Pull request by JessicaDJ0807**
BUG: 301040921

### CODE REVIEW

**Review by BaiiYuan — APPROVED** [review]
LGTM. Let's wait approval from other reviewers.

File: common/dpsl.ts [hunk 1]
```diff
@@ -355,7 +355,7 @@ export interface RunSmartctlCheckRequest {
   percentage_used_threshold?: number;
 }
 
-export type DpslTypes =
+export type TelemetryUnion =
```
Comment (ycheng627) [inline common/dpsl.ts:358]: I'm not sure what this is a union of by the naming.

Can we be more specific? 

If it is the possible telemetry types (e.g. cpu, memory, etc), lets call it `TelemetryTypeUnion`

if it is the possible telemetry results (e.g. CpuResult, MemoryResult), lets call it `TelemetryResultUnion`

Comment (JessicaDJ0807) [reply common/dpsl.ts]: Since it is the union of some telemetry info, it is renamed to TelemetryInfoUnion.

**Review by ycheng627 — APPROVED** [review]
LGTM

**Pull request merged**